index.html

```diff
@@ -2001,44 +2001,75 @@ <h3>Securing Mechanisms</h3>
 
         <p>
 It is expected that securing mechanism specifications other than the ones
-referred to above will be created as necessary. A securing mechanism
-specification:
+referred to above will be created as necessary.
         </p>
 
-        <ul>
-          <li>
-MUST document normative algorithms that provide content integrity protection
-for <a>conforming documents</a>. The algorithms MAY be general in nature and
-MAY be used to secure data other than <a>conforming documents</a>.
-          </li>
-          <li>
-MUST provide a verification mechanism that returns only the information in the
-<a>conforming document</a> that has been secured. It MAY provide an
-additional mechanism to receive other information that might be helpful
-during validation or for debugging purposes. The concrete interface that is
-expected to be provided is detailed in the
-<a href="#verify-securing-mechanism">securing mechanism verification
-algorithm</a> section.
-          </li>
-          <li>
-MUST document post-processing considerations for data returned from verification
-processes. This includes, but is not limited to, what information has been
-secured, for what period of time after verification, when or what additional
-information might be required to assist in decision making, and when
-care is necessary even when the returned data was properly secured.
-          </li>
-          <li>
-SHOULD provide integrity protection for any information referenced by a URL that
-is critical to validation. Mechanisms that can achieve this protection are
-discussed in Section <a href="#integrity-of-related-resources"></a> and Section
+        <p>
+Securing mechanism specifications MUST document normative algorithms that
+provide content integrity protection for <a>conforming documents</a>. The
+algorithms MAY be general in nature and MAY be used to secure data other than
+<a>conforming documents</a>.
+        </p>
+
+        <p>
+Securing mechanism specifications MUST provide a verification mechanism that
+returns only the information in the <a>conforming document</a> that has been
+secured, without any securing mechanism information included, such as `proof` or
+JOSE/COSE metadata. Specifications MAY provide additional mechanisms to convey
+other information that might be helpful during validation or for debugging
+purposes, such as securing mechanism metadata. A securing mechanism's
```

Comment:
To clarify that information can only be helpful for more than just the given two examples:

```suggestion
JOSE/COSE metadata. Specifications MAY provide additional mechanisms to convey
other information that might be helpful (for example, during validation or for debugging
purposes), such as securing mechanism metadata. A securing mechanism's
```